src/main/java/org/kainos/ea/auth/TokenService.java

```diff
@@ -0,0 +1,38 @@
+package org.kainos.ea.auth;
+
+import com.auth0.jwt.exceptions.JWTCreationException;
+import org.kainos.ea.client.FailedToGenerateTokenException;
+import org.kainos.ea.client.FailedToGetUserId;
+import org.kainos.ea.db.AuthDao;
+
+import java.sql.SQLException;
+
+public class TokenService {
+    private AuthDao authDao;
+    private JWTService jwtTokenService;
+
+
+    public TokenService(AuthDao authDao, JWTService jwtTokenService)
+    {
+        if(authDao == null || jwtTokenService == null)
+        {
+            throw new NullPointerException("Null service passed into constructor.");
+        }
```

Comment:
Put a similar comment on Leo's UI. Can you, as a team, come up with a consistent way of handling these null checks? I don't mind which way you go as long as it's consistent